movie_bet_bot/models/commands/commands.py

```diff
@@ -0,0 +1,59 @@
+import discord
+from movie_bet_bot.models.logger.logger import Logger
+
+# creates the help command. Always add this command last
+def create_help(bot):
+    # sets the name of the command
+    name = "help"
+    # sets the description of the command
+    description: str = "Shows a list of all commands."
+
+    # creates the help command
+    @bot.command_tree.command(name = name, description = description)
+    # the function that is called when the command is executed
+    async def help(interaction: discord.Interaction):
+        Logger.info("Running Help Command")
+        # creates a new embed
+        embed: discord.Embed = discord.Embed(description = "List of all commands:", colour = 0x000000)
```

Comment:
`color` is aliased to `colour` - just use that

```suggestion
        embed: discord.Embed = discord.Embed(description = "List of all commands:", color = 0x000000)
```